src/rpc.rs

```diff
@@ -237,10 +243,6 @@ impl Connection {
         let tx = hex::decode(&tx).chain_err(|| "non-hex tx")?;
         let tx: Transaction = deserialize(&tx).chain_err(|| "failed to parse tx")?;
         let txid = self.query.broadcast(&tx)?;
-        self.query.update_mempool()?;
```

Comment:
Doesn't removing it mean that you will not get an immediate notification, but wait for the periodic 5 seconds?
Maybe we can get the changed txids in `update_mempool`, and then call `on_scripthash_change` after parsing the transactions?...

Reply:
Yes, but with PeriodicUpdate taking 3+ seconds to begin with, there isn't much difference. Also, if within the scope of this project, I want to add optional support for ZMQ, which would remove the 5 second delay.

Reply:
from UX perspective - getting the sender's tx indexed is important for fetching history and unspent immediately after the transfer.
I think there's a non-negligible difference between 3+ seconds (I think it's less actually when you force a mempool-only update not within the 5 seconds interval) and waiting this _plus_ waiting for the interval to occur (8 seconds?)...